agent/acs/handler/acs_handler.go

```diff
@@ -264,6 +264,20 @@ func (acsSession *session) startACSSession(client wsclient.ClientServer, timer t
 
 	client.AddRequestHandler(refreshCredsHandler.handlerFunc())
 
+	// Add handler to ack ENI attach message
+	eniAttachHandler := newAttachENIHandler(
```

Comment:
This also needs a unit test in acs_handler_test.go. You need to modify https://github.com/aws/amazon-ecs-agent/blob/master/agent/acs/handler/acs_handler_test.go#L53 with mock values for the new fields that we expect and ensure that the handler is invoked.